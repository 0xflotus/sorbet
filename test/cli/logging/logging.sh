LOG_FILE=$(mktemp)
main/ruby-typer -e '1' -q --debug-log-file="$LOG_FILE"
echo LOG BEGINS
# only keep message parts and drop timings
grep -v "ms" "$LOG_FILE" | grep -v debug-log-file
