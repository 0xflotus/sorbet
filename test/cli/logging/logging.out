LOG BEGINS
Creating 1 worker threads
Worker threads created
Multiplexing job
Worker got task
Multiplexing job
Worker got task
Multiplexing job
Worker got task
Killing worker thread
